ambry-store/src/test/java/com.github.ambry.store/StorageManagerTest.java

```diff
@@ -83,22 +91,15 @@ public void mountPathNotFoundTest() throws Exception {
         downReplicaCount++;
       }
     }
-    deleteDirectory(new File(mountPathToDelete));
-    StorageManager storageManager = createAndStartStoreManager(replicas, metricRegistry);
+    Utils.deleteFileOrDirectory(new File(mountPathToDelete));
+    StorageManager storageManager = createStorageManager(replicas, metricRegistry);
+    storageManager.start();
     Map<String, Counter> counters = metricRegistry.getCounters();
+    assertEquals(0,
```

Comment:
minor: include a message in these asserts

Reply:
added